src/Npgsql/PostgresDatabaseInfo.cs

```diff
@@ -82,6 +82,11 @@ class PostgresDatabaseInfo : NpgsqlDatabaseInfo
         /// </summary>
         public virtual bool HasEnumSortOrder => Version >= new Version(9, 1, 0);
 
+        /// <summary>
+        /// True if the 'pg_type' table includes the 'typcategory' column; otherwise, false.
+        /// </summary>
+        public virtual bool HasTypCategory => Version >= new Version(8, 4, 0);
```

Comment:
Let's change this to `HasTypeCategory`. PostgreSQL shortens the word...but we can afford the extra letter :)